src/modules/load_mon/LoadMon.cpp

```diff
@@ -211,6 +246,8 @@ usage and publish the `cpuload` topic.
 
 On NuttX it also checks the stack usage of each process and if it falls below 300 bytes, a warning is output,
 which will also appear in the log file.
+
+On Linux stack check is disabled.
```

Comment:
Probably better to say it's only support on NuttX (there's also MacOS and QuRT).